script/Ciel/Day 7/07_01_CIEL07_2_2.txt

```diff
@@ -645,37 +689,40 @@ It's an enchanting whisper, yet somewhat desolate.
 [sha:ade496559e532232b2861c68e367e3a9c69207d6]{
 -- Page 326, Offset 15596. Mods: @n.
 -- 「予定表。わたしね、子供の頃からリストを作るのが趣味なの。
-"My itinerary. See, one of my hobbies since I was a kid has been making lists.
+"My itinerary. See, ever since I was a kid one of my hobbies has been making lists.
 }
 [sha:098dd3ceadb8c0ed604f8dde215c0c20e2864362]{
 -- Page 326, Offset 15597. Glued.
 -- やりたい事、やらない事、欲しいモノ、要らないモノ。そういうものを順序立ててメモっていくの」
-Want I want to do, what I don't want to do, things I need, things I don't need. I memo down stuff like this――put them in order."
+ What I want to do, what I don't want to do, things I need, things I don't need.
+ I jot down stuff like this to put it in order."
 }
 [sha:12722a77c6107793b27785c1a559f47c81c5391b]{
 -- Page 327, Offset 15598.
 -- 「授業中にイラッときたからお仕置きするリスト、
-"The list of people I'll punish if I get angry in class,
+"The list of people I'll punish for pissing me off during class.
 }
 [sha:361dda3896672b8082d887953803f3c2c9223fe8]{
 -- Page 327, Offset 15599.
 -- 親切にしてもらったから恩返しするリスト、
-The list of people who I'll repay for being kind to me,
+The list of people who I'll repay for being kind to me.
 }
 [sha:7a47e735c78530cef4ca2fefcde610a816fdfafe]{
 -- Page 327, Offset 15600.
 -- いやらしい教頭先生にブチかます社会的制裁リスト、エトセトラエトセトラ」
-The list of societal punishments I'd doll out on those disgusting teachers above me, et cetera, et cetera."
+// Is 社会的制裁 commonish parlance in JP? Fringe term in EN
+The list of social sanctions I'll dole out to that disgusting vice principal, et cetera, et cetera."
```

Comment:
On reflection, this is probably just a singular 教頭, but because there are often (or rather, it's required for there to be) more in larger schools, I assumed several, as that is what I am used to. Shiki's school seems rather small though, so who knows.